client/js/test/fixtures/sheet-container.html

```diff
@@ -12,5 +12,11 @@
 
       </tr>
     </tbody>
+
+    <tfoot>
+      <tr>
+
```

Comment:
No need for extra blank lines between opening and closing tag. It breaks up the visual flow in a way that suggests that you WERE going to write code there and then forgot to.